### ISSUE TITLE
Update systemd tests to run on Travis's new Ubuntu images

### ISSUE THREAD

**Issue by halter73** [aspnet/KestrelHttpServer#1912]
We should have until September 1st before the deprecated Travis images are no longer available.

https://github.com/aspnet/KestrelHttpServer/pull/1911#issuecomment-310175096

**Comment by halter73** [comment]
The systemd tests fail on Travis even with the [deprecated image](https://github.com/aspnet/KestrelHttpServer/blob/3084227314c784e34c6ca19402346bc0eafce2ea/.travis.yml#L32). I am going to disable the systemd tests temporarily so the travis badge can go back to green.

From [Travis failure logs](https://travis-ci.org/aspnet/KestrelHttpServer/jobs/246447375):
```
The command "./build.sh" exited with 0.
63.16s$ if test "$TRAVIS_OS_NAME" != "osx"; then bash test/Microsoft.AspNetCore.Server.Kestrel.FunctionalTests/SystemdActivation/docker.sh; fi
Copyright (C) Microsoft Corporation. All rights reserved.
e/travis/build/aspnet/KestrelHttpServer/src/Microsoft.AspNetCore.Server.Kestrel.Transport.Abstractions/bin/Debug/netstandard2.0/Microsoft.AspNetCore.Server.Kestrel.Transport.Abstractions.dll
e/travis/build/aspnet/KestrelHttpServer/src/Microsoft.AspNetCore.Server.Kestrel.Core/bin/Debug/netstandard2.0/Microsoft.AspNetCore.Server.Kestrel.Core.dll
  Microsoft.AspNetCore.Server.Kestrel.Transport.Libuv -> /home/travis/build/aspnet/KestrelHttpServer/src/Microsoft.AspNetCore.Server.Kestrel.Transport.Libuv/bin/Debug/netstandard2.0/Microsoft.AspNetCore.Server.Kestrel.Transport.Libuv.dll
e/travis/build/aspnet/KestrelHttpServer/src/Microsoft.AspNetCore.Server.Kestrel/bin/Debug/netstandard2.0/Microsoft.AspNetCore.Server.Kestrel.dll
  Microsoft.AspNetCore.Server.Kestrel.Https -> /home/travis/build/aspnet/KestrelHttpServer/src/Microsoft.AspNetCore.Server.Kestrel.Https/bin/Debug/netstandard2.0/Microsoft.AspNetCore.Server.Kestrel.Https.dll
pleApp -> /home/travis/build/aspnet/KestrelHttpServer/samples/SampleApp/bin/Debug/netcoreapp2.0/SampleApp.dll
  SampleApp -> /home/travis/build/aspnet/KestrelHttpServer/samples/SampleApp/bin/Debug/netcoreapp2.0/publish/
Sending build context to Docker daemon 537.1 MB
Step 1 : FROM microsoft/dotnet-nightly:2.0-runtime-deps
2.0-runtime-deps: Pulling from microsoft/dotnet-nightly
c75480ad9aaf: Pulling fs layer
ee23f0117f66: Pulling fs layer
ee23f0117f66: Verifying Checksum
ee23f0117f66: Download complete
c75480ad9aaf: Verifying Checksum
c75480ad9aaf: Download complete
c75480ad9aaf: Pull complete
ee23f0117f66: Pull complete
Digest: sha256:42400a9a449be9171977768c099a81e822178998daeaf462fadbda5014f32128
Status: Downloaded newer image for microsoft/dotnet-nightly:2.0-runtime-deps
 ---> 6ff20765eff5
Step 2 : ENV container docker
 ---> Running in 1f247b6f161e
 ---> 2d711b2d2846
Removing intermediate container 1f247b6f161e
Step 3 : VOLUME /sys/fs/cgroup
 ---> Running in c1c85fd38dc3
 ---> 944554bd0b4a
Removing intermediate container c1c85fd38dc3
Step 4 : EXPOSE 8080
 ---> Running in 569d008a9223
 ---> 9abcadc52131
Removing intermediate container 569d008a9223
Step 5 : ADD .dotnet/ /usr/share/dotnet/
 ---> cde0e03e9c6c
Removing intermediate container 01ba97b7c6be
Step 6 : ADD publish/ /publish/
 ---> 52b220c17c06
Removing intermediate container a9dc4678dd28
Step 7 : ADD activate-kestrel.socket /etc/systemd/system/activate-kestrel.socket
 ---> 61c69b12cd08
Removing intermediate container 91a36a847197
Step 8 : ADD activate-kestrel.service /etc/systemd/system/activate-kestrel.service
 ---> 9a3c4fdf2a7a
Removing intermediate container 48472a9649b8
Step 9 : RUN apt-get -o Acquire::Check-Valid-Until=false update
 ---> Running in 4e662d5cc570
Get:1 http://security.debian.org stretch/updates InRelease [62.9 kB]
Get:2 http://security.debian.org stretch/updates/main amd64 Packages [61.1 kB]
Ign:3 http://deb.debian.org/debian stretch InRelease
Get:4 http://deb.debian.org/debian stretch-updates InRelease [88.5 kB]
Get:5 http://deb.debian.org/debian stretch Release [113 kB]
Get:6 http://deb.debian.org/debian stretch Release.gpg [3108 B]
Get:7 http://deb.debian.org/debian stretch/main amd64 Packages [9497 kB]
Fetched 9826 kB in 1s (5383 kB/s)
Reading package lists...
 ---> 358a8d74132b
Removing intermediate container 4e662d5cc570
Step 10 : RUN apt-get install -y dbus
 ---> Running in 038c14fcbc9d
Reading package lists...
Building dependency tree...
Reading state information...
The following additional packages will be installed:
  libapparmor1 libdbus-1-3 libexpat1
Suggested packages:
  default-dbus-session-bus | dbus-session-bus
The following NEW packages will be installed:
  dbus libapparmor1 libdbus-1-3 libexpat1
0 upgraded, 4 newly installed, 0 to remove and 0 not upgraded.
Need to get 563 kB of archives.
After this operation, 1522 kB of additional disk space will be used.
Get:1 http://deb.debian.org/debian stretch/main amd64 libapparmor1 amd64 2.11.0-3 [78.7 kB]
Get:2 http://deb.debian.org/debian stretch/main amd64 libdbus-1-3 amd64 1.10.18-1 [192 kB]
Get:3 http://deb.debian.org/debian stretch/main amd64 libexpat1 amd64 2.2.0-2 [83.3 kB]
Get:4 http://deb.debian.org/debian stretch/main amd64 dbus amd64 1.10.18-1 [208 kB]
debconf: delaying package configuration, since apt-utils is not installed
Fetched 563 kB in 0s (4822 kB/s)
Selecting previously unselected package libapparmor1:amd64.
(Reading database ... 7094 files and directories currently installed.)
Preparing to unpack .../libapparmor1_2.11.0-3_amd64.deb ...
Unpacking libapparmor1:amd64 (2.11.0-3) ...
Selecting previously unselected package libdbus-1-3:amd64.
Preparing to unpack .../libdbus-1-3_1.10.18-1_amd64.deb ...
Unpacking libdbus-1-3:amd64 (1.10.18-1) ...
Selecting previously unselected package libexpat1:amd64.
Preparing to unpack .../libexpat1_2.2.0-2_amd64.deb ...
Unpacking libexpat1:amd64 (2.2.0-2) ...
Selecting previously unselected package dbus.
Preparing to unpack .../dbus_1.10.18-1_amd64.deb ...
Unpacking dbus (1.10.18-1) ...
Setting up libexpat1:amd64 (2.2.0-2) ...
Processing triggers for libc-bin (2.24-11+deb9u1) ...
Setting up libapparmor1:amd64 (2.11.0-3) ...
Setting up libdbus-1-3:amd64 (1.10.18-1) ...
Setting up dbus (1.10.18-1) ...
invoke-rc.d: could not determine current runlevel
invoke-rc.d: policy-rc.d denied execution of start.
Processing triggers for libc-bin (2.24-11+deb9u1) ...
 ---> 8269c5c724bb
Removing intermediate container 038c14fcbc9d
Step 11 : RUN systemctl mask getty.target console-getty.service
 ---> Running in 415437249c02
invalid header field value "oci runtime error: container_linux.go:247: starting container process caused \"exec: \\\"systemctl\\\": executable file not found in $PATH\"\n"
The command "if test "$TRAVIS_OS_NAME" != "osx"; then bash test/Microsoft.AspNetCore.Server.Kestrel.FunctionalTests/SystemdActivation/docker.sh; fi" exited with 1.
Done. Your build exited with 1.
```

**Comment by halter73** [comment]
Fixed by #1925. Thanks again @onyxmaster.
